src/wallet/wallet.cpp

```diff
@@ -2956,6 +2953,9 @@ bool CWallet::CreateTransactionInternal(
                     txNew.vout.push_back(txout);
                 }
 
+                // Calculate the fees for things that aren't inputs, excluding the change output
+                const CAmount not_input_fees = coin_selection_params.m_effective_feerate.GetFee(coin_selection_params.tx_noinputs_size);
```

Comment:
In commit "Roll not_input_fees into nValueToSelect instead of having it be separate" (6bd07ddff8c092ac57fc72e47510c73864ac7acc)

Not necessarily relevant to this commit, but there is a big block of code above that just seems to be filling the coin_selection_params struct. It could be good for a future cleanup to move that code into a function to be able to shrink this function and declare coin_selection_params as const